api/apiclient_test.go

```diff
@@ -60,19 +61,19 @@ func (s *apiclientSuite) TestConnectWebsocketMultiple(c *gc.C) {
 
 	// Check that we can use the proxy to connect.
 	info.Addrs = []string{proxy.Addr()}
-	conn, _, err := api.ConnectWebsocket(info, api.DialOpts{})
+	conn, _, err := api.DialAPI(info, api.DialOpts{})
 	c.Assert(err, jc.ErrorIsNil)
 	conn.Close()
-	assertConnAddrForEnv(c, conn, proxy.Addr(), s.State.ModelUUID(), "/api")
+	assertConnAddrForModel(c, conn, proxy.Addr(), s.State.ModelUUID())
```

Comment:
This is an existing test that I've just changed because I renamed some things. If we want to fix it, it would be better to do in another PR, if that's OK.